### DISCUSSION TITLE
Connecting Designs Addon and Figma Plugin

### DISCUSSION THREAD

**Discussion by thafryer** [storybookjs/addon-designs#183] [Ideas]
Hi Pocka! I’m Jarel, an Engineering Manager at Chromatic. We would love to collaborate with you to create a seamless connection between the Figma plugin and this addon so that users can reuse their existing story to design connections made via the Figma plugin. In addition, we’d also like to provide first-class support for storybook-addon-designs in the Storybook ecosystem. The work could include the following: 

- Bringing the addon into the Storybook GitHub and NPM organization
- Aligning with Storybook 7.0 Design Refresh.
- Enable design URLs to sync between the [Figma plugin ](https://www.figma.com/community/plugin/1056265616080331589)and the Designs addon
- Providing a higher degree of support for end users.

Please share with us your thoughts or interest in making this happen.

**Comment by pocka** [comment]
Sounds great!

I believe transfering the repo under Storybook Org is the first step we should take.
Can you provide me steps and/or requirements (repo name, docs changes, etc...) to do this?
I'm in GitHub Storybook Org (member) and am able to create a repo, so I could transfer the repo immediately if no changes has to be made.

**Reply by thafryer** [reply]
Awesome! Agreed as the first step to take! The only change to make in the ownership transfer is the name. Let's change the name to `addon-designs`.

**Reply by thafryer** [reply]
@pocka Let me know when you start the trasnfer.

**Reply by pocka** [reply]
I'll transfer the repo on Mar 11 (or 12, if something went wrong). My plan is below:

1. Disable GitHub Actions workflows by removing workflow files except [`check-pushes.yml`](https://github.com/pocka/storybook-addon-designs/blob/de0ce0ae8c6cb15be0d52bc696cac6893b546311/.github/workflows/check-pushes.yml)
    - In a single commit to be easy to rebase
2. Transfer the repo under Storybook org as `addon-designs`
3. Add a comment to this thread

**Reply by pocka** [reply]
Done. (workflows removal: c034db2d91a9f3f956b274441b9104bf5af5b99d)

**Comment by pocka** [comment]
Can you share the current status? What else can I do?

The repository transfer process is completed, but everything else is untouched.
Development & release workflow (CI/CD), docs/sample page hosting, design URL sync feature, 7.0 support, etc...

**Reply by thafryer** [reply]
Hey Pocka! My apologies for slow response here. Now that we are on 7.0 and have a small CI/CD workflow. The next pieces of work will be. I aim to write up a issue or GitHub project that we can use to work through these.
- Docs/Sample Page Hosting
- Aligning with Storybook 7.0 Design Refresh.
- Enable design URLs to sync between the [Figma plugin ](https://www.figma.com/community/plugin/1056265616080331589)and the Designs addon
